src/it/scala/iog/psg/cardano/jpi/CardanoJpiSpec.scala

```diff
@@ -109,8 +92,8 @@ class CardanoJpiSpec extends AnyFlatSpec with Matchers with Configure with Model
     println(s"WALLET $baseUrl")
     val aryLen = testWalletMnemonic.split(" ").length
     val aryLen2 = testWallet2Mnemonic.split(" ").length
-    println(s"WALLET 1 words ${aryLen} <-")
-    println(s"WALLET 2 words ${aryLen2} <-")
+    println(s"WALLET 1 words $aryLen <-")
+    println(s"WALLET 2 words $aryLen2 <-")
```

Comment:
These can go. My bad.

Reply:
(Well they were needed for debugging CI, but they are not needed anymore)

Reply:
ok will get rid of them